src/components/Executions/filters/useCurrentUserOnlyFilterState.ts

```diff
@@ -0,0 +1,43 @@
+import { FilterOperationName } from 'models/AdminEntity/types';
+import { UserProfile } from 'models/Common/types';
+import { useState } from 'react';
+import { BooleanFilterState } from './types';
+import { useFilterButtonState } from './useFilterButtonState';
+import { useUserProfile } from 'components/hooks/useUserProfile';
+
+/** Maintains the state for a button to be used for filtering by user.
+ */
+function getUserId(profile: UserProfile): string {
+    return profile.sub ? profile.sub : '';
```

Comment:
This is different from what you were using before (checking preferredName and a different field). I'm not sure what `sub` is for or whether it's always the right value to use. @EngHabu do you know?

Reply:
Yeah, I used wrong value for filtering executions before. In this API request, we need `userId`, not `name`. In the above code, `sub` is the same with `userId`. 
So it is right value for filter.